src/api/AuthServer/AuthServer/Valdators/PasswordValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentValidation;
+
+public class PasswordValidator : AbstractValidator<string>
```

Comment:
Good idea to carry password validator out because it's used in 2 places, but it will be registered in the validation pipeline (`AddValidatorsFromAssemblyContaining`) and when we send any string to the web api, it will be validated as a password. Not the thing we expect it to be. Exclude it from the validation pipeline